### DISCUSSION TITLE
OpenDRT v0.0.82

### DISCUSSION THREAD

**Discussion by jedypod** [jedypod/open-display-transform#5] [General]
## Features
- Add input transfer function options with common camera log and log working space encodings for the Resolve DCTL. 
- Add Blackmagic Gen5 colorscience inputs: Blackmagic Wide Gamut // Blackmagic Film Gen5 Log encoding
- Add lenscap black subtraction compensation feature. Pretty experimental but it is intended to add a small value to compensate for negative grain on input. Since OpenDRT maps 0.0 to 0.0, this helps preserve shadow detail. Disable this if manually compensating black levels while grading.
- Add gamut compression operator to better handle extremely high purity input colors.
- Update perceptual dechroma to use ICtCp colorspace. This biases the chromaticity-linear hue paths of the highlight dechroma and gamut compression, along perceptual hue-lines, resulting in more natural looking colors and better appearance matching between HDR and SDR outputs.

<hr /><em>This discussion was created from the release <a href='https://github.com/jedypod/open-display-transform/releases/tag/v0.0.82'>OpenDRT v0.0.82</a>.</em>

**Comment by claxonberlin** [comment]
Hi Jed. Ever since I read up on Troy Sobotka's Filmic Blender I completely fell into the rabbit hole of colour management. It's horrible here, but very fascinating! I've been reading [the wonderfuly nerdy thread](https://community.acescentral.com/t/output-transform-tone-scale/3498/174) on ACEScentral with awe.

I was wondering what is needed to create an OCIO config for OpenDRT. Is there any chance of getting this into Blender, Affinity Photo or other DCC packages?

**Reply by jedypod** [reply]
Hey @clausbertels, thanks for the comment! Yes I've been in this rabbit hole for quite a while now and I don't think I'll be getting out anytime soon haha!

Your comment has good timing, I was actually starting on putting together an ocio config with opendrt this afternoon. It's been on my todo list for a long time, I just keep getting distracted by more interesting problems. It's not hard to do, will just take a bit of time. I created [an issue here](https://github.com/jedypod/open-display-transform/issues/17) so we can track progress.

Stay tuned!

**Reply by jedypod** [reply]
Just following up: An OCIO config was [released with OpenDRT v1.0.0](https://github.com/jedypod/open-display-transform/releases/tag/v1.0.0)
